### ISSUE TITLE
Empty container topic and its child topics are not shown in Website format

### ISSUE THREAD

**Issue by claudiusaiz** [EWSoftware/SHFB#1016]
Hello,

I am trying to migrate an old project from the "HTML Help 1" format to the "Website" format. 

In the previous format, there was an empty container topic containing two children non-empty topics. Afterwards followed the namespace documentation. Here is how the resulting chm file looked like:
![help1](https://github.com/EWSoftware/SHFB/assets/1729434/6ae77265-9b1e-445d-818c-01ed0462adf0)

When switching the format to "Website", the empty container topic and its children are not shown anymore:
![website](https://github.com/EWSoftware/SHFB/assets/1729434/cd895065-b7e0-457d-8137-08cb3e656614)

Is there any extra configuration I should do so that the website format looks the same?

I am using SHFB version 2023.7.8.0.

I've also attached an archive with all the files used in the project.

Thanks!

[projectfiles.zip](https://github.com/EWSoftware/SHFB/files/12309840/projectfiles.zip)

**Comment by EWSoftware** [comment]
Load the site using the View Website option so that it uses a local web server instance.  If you load it directly from the file system, the browser blocks the scripts so the TOC will not display correctly.

**Comment by claudiusaiz** [comment]
Hm, I installed IIS 10.0 Express, restarted everything and then clicked on "View Website (Local Web Dev Server)". It launched from a local server but the result is still the same:
![website_server](https://github.com/EWSoftware/SHFB/assets/1729434/5d53c08b-17c5-499b-83d3-700e6ed4ddee)

Is there any server configuration I should do? Or other ideas on how I should proceed next?

Thanks!

**Comment by EWSoftware** [comment]
The root node looks like it's collapsed.  Have your tried expanding it?

**Comment by claudiusaiz** [comment]
Thanks, that works. Is there a way to keep the root node expanded? I notice that every time I click on a topic, the root node collapses again.

Also, there is still a bit of strange behavior - after clicking on one of the child topics and re-expanding the root node, the empty container topic is duplicated:
![strange](https://github.com/EWSoftware/SHFB/assets/1729434/47243b68-c627-41de-b073-cb12517f788a)

**Comment by EWSoftware** [comment]
No, it's not possible to keep them expanded.  Regarding the bug, it's not one I'm likely to fix since it's the old presentation style and is only there for supporting the older help 1 and help viewer formats.  You might try the new Default 2022 style which is the most recent.

**Comment by claudiusaiz** [comment]
Ok, will try the new presentation style, thank you for your help.
